components/builder-api-client/src/error.rs

```diff
@@ -54,11 +54,11 @@ impl fmt::Display for Error {
             Error::APIError(ref c, ref m) if !m.is_empty() => format!("[{}] {}", c, m),
             Error::APIError(ref c, _) => format!("[{}]", c),
             Error::BadResponseBody(ref e) => format!("Failed to read response body, {}", e),
-            Error::DownloadWrite(ref p, ref e) => format!(
-                "Failed to write contents of builder response, {}, {}",
-                p.display(),
-                e
-            ),
```

Comment:
This is an example of where I think `force_multiline_blocks = true` results in more readable code.